src/components/Button/index.tsx

```diff
@@ -0,0 +1,32 @@
+type Props = {
+  color?: string;
+  border?: string;
+  children?: React.ReactNode;
+  height?: string;
+  onClick?: () => void;
+  radius?: string;
+  width?: string;
+  textColor?: string;
+};
+
+const Button: React.FC<Props> = ({
```

Comment:
Should we use `React.FC` to declare the argument type for the function?
I know the React team removed the children prop in the `React.FC` type.